code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -98,8 +98,6 @@
 			podman.ckey = ckey_holder
 		podman.gender = blood_gender
 		podman.faction |= factions
-		podman.faction |= "plants"
-		podman.faction |= "vines"   //Pod grown Diona are allied with plants and vines alike.
```

Comment:
Might wanna document that

Reply:
This is actually handled on the podman species subtype.

When Diona were reworked, there's a subtype that heals, even in crit and also has these special factions added on species gain and losts when they change species.

No need for this to be here anymore.